packages/cli/src/commands/cliExecuteSwap.ts

```diff
@@ -86,6 +94,11 @@ export default async function cliExecuteSwap(
       destAsset: args.destAsset,
       amount: args.amount,
       destAddress: args.destAddress,
+      ccmMetadata: args.message &&
```

Comment:
maybe throw an error if one of them is missing like we do in the `cliRequestSwapDepositAddress`